src/Indicator.vala

```diff
@@ -18,15 +18,31 @@
 public class BluetoothIndicator.Indicator : Wingpanel.Indicator {
     public bool is_in_session { get; construct; default = false; }
 
-    private Widgets.PopoverWidget popover_widget;
-    private Widgets.DisplayWidget dynamic_icon;
+    public signal void device_requested (BluetoothIndicator.Services.Device device);
+    private BluetoothIndicator.Services.Device paired_device;
+    BluetoothIndicator.Widgets.PopoverWidget popover_widget;
```

Comment:
That's fine to be more specific about namespace, but we should still explicitly define scope:
```suggestion
    private BluetoothIndicator.Widgets.PopoverWidget popover_widget;
```